bootstrap.sh

```diff
@@ -1 +1,21 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
+kubectl create namespace mysql
+
+# Apply MySQL StatefulSet and ConfigMap
+kubectl apply -f .infrastructure/mysql/mysql-secret.yml
+kubectl apply -f .infrastructure/mysql/init-configMap.yml
+kubectl apply -f .infrastructure/mysql/mysql-headless-service.yml
+kubectl apply -f .infrastructure/mysql/statefulSet.yml
+
+#Create todoaapp and services
+kubectl apply -f .infrastructure/pv.yml
+kubectl apply -f .infrastructure/pvc.yml
+kubectl apply -f .infrastructure/clusterIp.yml
+kubectl apply -f .infrastructure/configMap.yml
```

Comment:
There is a typographical error in the file name `.infrastructure/configMap.yml`. Ensure that it matches the correct file name in your directory structure.